WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtils.java

```diff
@@ -245,6 +245,14 @@ public void onClick(View v) {
             } else {
                 int messageRes = post.isPage() ? R.string.page_published : R.string.post_published;
                 UploadUtils.showSnackbar(snackbarAttachView, messageRes);
+                UploadUtils.showSnackbarSuccessAction(snackbarAttachView, messageRes,
+                        R.string.button_view, new View.OnClickListener() {
+                            @Override
+                            public void onClick(View view) {
+                                // jump to Editor Preview mode to show this Post
+                                ActivityLauncher.viewPostPreviewForResult(activity, site, post);
```

Comment:
This is inconsistent with the `view` button in the post list, which uses `ActivityLauncher.browsePostOrPage()` to show the page in a WebView.

Reply:
good observation! 🙇 addressed in c486c17